src/material/snack-bar/snack-bar-container.scss

```diff
@@ -10,13 +9,28 @@
   @include mdc-snackbar.static-styles($query: mdc-helpers.$mdc-base-styles-without-animation-query);
 }
 
+@mixin _container-min-width {
+  $min-width: mdc-snackbar-theme.$min-width;
+  $mobile-breakpoint: mdc-snackbar-theme.$mobile-breakpoint;
+
+  // The styles weren't included in `static-styles` so we need to add them ourselves.
+  @include mdc-snackbar-theme.min-width(
+    $min-width: $min-width,
+    $mobile-breakpoint: $mobile-breakpoint,
+    $query: mdc-helpers.$mdc-base-styles-query
+  );
+
+  // The MDC `min-width` mixin has a similar breakpoint that sets `min-width: 100%` on the surfcace
+  // element to make it span the entire viewport, however it ends up collapsing because the
+  // container is `width: auto`. This query ensures that the surface will span the whole viewport.
```

Comment:
Should we open an issue OR code change with MDC for this? It seems like `100vw` more closely matches the intent of their style

Reply:
MDC would've been 100vw as well, but we override their `position: fixed` positioning in order to fit the snack bar in our overlay positioning model.